addons/maverick/CfgAmmo.hpp

```diff
@@ -6,7 +6,7 @@ class CfgAmmo {
     class Missile_AGM_02_F : MissileBase {};
     
     class GVAR(L) : Missile_AGM_02_F {
-        author = "rufix";
+        author = "xrufix";
```

Comment:
so close

Reply:
Question: Should i remove my name and replace it with `ECSTRING(common,ACETeam);`?